### ISSUE TITLE
Optipng-bin now used ES modules

### ISSUE THREAD

**Issue by bramstein** [papandreou/node-optipng#89]
I was just upgrading to the latest version of assetgraph-builder which pulls in this package. It looks like the optipng-bin package switched to using ES module syntax and this package is broken:

* https://github.com/imagemin/optipng-bin/pull/120
* https://github.com/papandreou/node-optipng/pull/88

```
Unhandled rejection Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /home/node/.npm-global/lib/node_modules/assetgraph-builder/node_modules/optipng-bin/index.js
require() of ES modules is not supported.
require() of /home/node/.npm-global/lib/node_modules/assetgraph-builder/node_modules/optipng-bin/index.js from /home/node/.npm-global/lib/node_modules/assetgraph-builder/node_modules/optipng/lib/OptiPng.js is an ES module file as it is a .js file whose nearest parent package.json contains "type": "module" which defines all .js files in that package scope as ES modules.
Instead rename index.js to end in .cjs, change the requiring code to use import(), or remove "type": "module" from /home/node/.npm-global/lib/node_modules/assetgraph-builder/node_modules/optipng-bin/package.json.

```

**Comment by papandreou** [comment]
Damn, looks like the test suite only exercised the case where a system-wide `optipng` is installed on CI. Added test coverage in 7e62150af01495f816a2cafd8596a02812064587, downgraded `optipng-bin` to a non-ESM version, then released 4.1.0 and ag-b 9.1.1 😇

**Comment by bramstein** [comment]
Amazing, I owe you 🍻 Thanks!
